x-pack/plugins/alerting/server/rules_client/lib/validate_actions.test.ts

```diff
@@ -283,4 +307,19 @@ describe('validateActions', () => {
       '"Failed to validate actions due to the following error: Action\'s alertsFilter days has invalid values: (111:[0,8]) "'
     );
   });
+
+  // TO DO talk to Christos
```

Comment:
This test was mostly to ensure that we do not perform the validations we do for normal actions to the system actions. I think we can remove it.